### DISCUSSION TITLE
Re-access the lost account

### DISCUSSION THREAD

**Discussion by reubenellisp** [community/community#11879] [Copilot Conversations]
Hi there, unfortunately, I have lost my GitHub account, and it was kind of an unwanted action. Thus, it caused the loss of access to the copilot services.
Is there anything that I can do to get back my access?

**Comment by davecheney** [comment]
What was your previous GitHub account name?

**Reply by reubenellisp** [reply]
ReubenEllis
I also have recreated the account with that username.

**Reply by davecheney** [reply]
Please give me 24-48 hours

**Reply by reubenellisp** [reply]
Okay, thank you, I really appreciate your effort.

**Reply by reubenellisp** [reply]
It seems like this account that I'm using now is flagged. Does that interrupt this process?

**Reply by davecheney** [reply]
Yes, you will have to resolve your status with support before I can continue.

**Reply by reubenellisp** [reply]
I have already opened a ticket, and I'm waiting for support.
I thought you might be unable to contribute to this discussion because it was not shown in the discussions list, so I created #11932.
thank you for your time.

**Reply by reubenellisp** [reply]
Hi there, the problem with my account is resolved now, so hopefully, you'll be able to address my OAuth issues.

**Reply by davecheney** [reply]
I've reactivated your copilot invite, you should have access to copilot now.
